src/server.rs

```diff
@@ -584,13 +626,27 @@ impl RollupBoostServer {
 
             let builder = self.builder_client.clone();
 
```

Comment:
Leaving this commented for now since it seems we have get_payload hardcoded currently.